packages/web3/src/constants.ts

```diff
@@ -22,7 +22,7 @@ export const MIN_UTXO_SET_AMOUNT = BigInt(1000000000000)
 export const ALPH_TOKEN_ID = ''.padStart(64, '0')
 export const ONE_ALPH = 10n ** 18n
 export const DUST_AMOUNT = 10n ** 15n
-export const ZERO_ADDRESS = 'tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq'
+export const NULL_CONTRACT_ADDRESS = 'tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq'
```

Comment:
Perhaps we could add a new `NULL_CONTRACT_ADDRESS` constant, which would avoid breaking compatibility with existing code, WDYT?